docs/tbdev_getting_started.ipynb

```diff
@@ -197,8 +197,7 @@
         "id": "5QH5k4AUNE27"
       },
       "source": [
-        "Each individual upload has a unique experiment ID. This means that if you start a new upload with the same directory, you will get a new experiment ID.\n",
-        "You can list all the experiments you have uploaded using \n",
+        "Each individual upload has a unique experiment ID. This means that if you start a new upload with the same directory, you will get a new experiment ID. You can view all the experiments you have uploaded in Tensorboard.dev experiment list at https://tensorboard.dev/experiments/. Alternatively, you can list your experiments in the terminal using the following command:\n",
```

Comment:
Suggestion (up to you). I think you can shorten:

You can view all the experiments you have uploaded in Tensorboard.dev experiment list at https://tensorboard.dev/experiments/.

To:

You can view all your uploaded experiments at https://tensorboard.dev/experiments/.

Reply:
Reworded. Thanks!